leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -1007,4 +1007,87 @@ private void closeKeyboard() {
       imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
     }
   }
+
+  public static class Builder {
+    private Double locationLatitude;
+    private Double locationLongitude;
+    private String locationSearchZone;
+    private String layoutsToHide = "";
+    private boolean enableSatelliteView = true;
+    private boolean shouldReturnOkOnBackPressed = false;
+    private List<LekuPoi> lekuPois;
+
+    public Builder() {
+    }
+
+    public Builder withLatitude(double latitude) {
```

Comment:
It's only to expose bundle params (that neither ensure to fill the two parameters), but ok, we can unify the Builder setter `withLocation(double latitude, double longitude)`